apps/docs/spec/supabase_js_v2.yml

```diff
@@ -4991,6 +4991,29 @@ functions:
             method: 'DELETE'
           })
           ```
+      - id: regional-invocation
+        name: Invoking a Function in the UsEast1 region
+        description: |
+          Here are the available regions:
+          - `FunctionRegion.Any`, `FunctionRegion.ApNortheast1`,
```

Comment:
I'm a bit confused by the pairing up of regions here, was expecting pairs to be somehow connected to each other but as far as I can tell, they're not. Could we just put every region as its own item instead?